omise/classes/charge.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
@nimid can you provide docblock for 
- create()
- getAmount()
- getCardToken()
- getCapture()
- getChargeDescription()
- getCurrencyCode()
- getSecretKey()